docs/topics/native/native-improving-compilation-time.md

```diff
@@ -72,6 +71,13 @@ Here are some recommendations for configuring Gradle for better compilation perf
   the build completes successfully. It is possible that these properties were added previously to work around issues that
   have already been fixed.
 
+* **Try incremental compilation of klib artifacts**. With incremental compilation, if your change of the project module
+  causes only a partial recompilation of source code into `klib` artifacts, just a part of the `klib` is further
+  recompiled into a binary.
+  
+  This feature is an [Experimental](components-stability.md#stability-levels-explained). To enable it,
+  add the `kotlin.incremental.native=true` option to your `gradle.properties` file.
```

Comment:
Should we explicitly mention that some bugs (e.g. compiler crashes) are possible, and encourage users to report them?